examples/streams-tracing/src/main.rs

```diff
@@ -85,7 +85,7 @@ async fn main() {
 
     let result = vixen::stream::Server::builder()
         .source(YellowstoneGrpcSource::new())
```

Comment:
```suggestion
let result = vixen::stream::Server::builder()
.source::<YellowstoneGrpcSource>()
```

To get where new is invoked internally but call new once the runtime has gathered the filters and we read the connection config we call `T::new(config, filters)`